app/src/main/java/com/getdefault/task9/DisplayFragment.kt

```diff
@@ -19,9 +22,18 @@ class DisplayFragment : Fragment() {
         val viewModel: DisplayViewModel = ViewModelProvider(this).get(DisplayViewModel::class.java)
         binding.viewModel = viewModel
 
-        val adapter = PersonAdapter()
+
+        val adapter = PersonAdapter { phone: String -> callPerson(phone) }
+
         binding.rvPersonList.adapter = adapter
 
+
         return binding.root
     }
+
+    private fun callPerson(phone : String){
+        val callIntent = Intent(Intent.ACTION_CALL)
```

Comment:
@saurav-kalsoor Might need to use Intent.ACTION_DIAL instead as Intent.ACTION_CALL requires permission from the user at runtime.

Reply:
Done sir